src/pages/Login.vue

```diff
@@ -33,32 +33,22 @@
         :isDisabled="!isRequestOtpEnabled"
       ></icon-button>
       <!-- button to submit OTP -->
-      <button
-        type="button"
-        :class="submitOTPButtonClass"
-        class="mt-2 flex justify-center items-center transition ease-in duration-200 text-center text-base font-semibold focus:shadow-none focus:outline-none focus:ring-2 focus:ring-offset-2 disabled:opacity-50 disabled:cursor-not-allowed"
-        :disabled="!isSubmitOTPEnabled || pending"
-        v-if="requestedOtp"
+      <icon-button
+        class="mt-2"
         @click="phoneLogin"
-      >
-        <div class="flex w-full justify-center">
-          <!-- text -->
-          <p :class="submitOTPTitleClass">{{ submitOTPTitle }}</p>
-          <!-- loading spinner -->
-          <inline-svg
-            v-if="pending"
-            :src="require('@/assets/images/spinner-solid.svg')"
-            class="animate-spin h-4 place-self-center ml-2 text-white"
-          ></inline-svg>
-        </div>
-      </button>
+        :titleConfig="submitOTPTitleConfig"
+        :iconConfig="submitOTPIconConfig"
+        :buttonClass="submitOTPButtonClass"
+        v-if="requestedOtp"
+        :disabled="!isSubmitOTPEnabled || submitOTPIconConfig.enabled"
+      ></icon-button>
       <!-- button to request resending OTP -->
       <icon-button
         @click="resendOtp"
         :titleConfig="resendOTPTitleConfig"
         :buttonClass="resendOTPButtonClass"
         class="mt-2"
-        :isDisabled="pending"
+        :isDisabled="pending || submitOTPIconConfig.enabled"

```

Comment:
pending is not needed here right? - had added during the same PR when adding spinner to submit OTP